.travis.yml

```diff
@@ -7,8 +7,8 @@ before-install:
 services:
   - docker
 
+martix:
```

Comment:
Should this be `matrix`? It is strange that the build isn't failing because of this...